earthpy/tests/test_stack.py

```diff
@@ -18,6 +18,45 @@ def in_paths(basic_image_tif):
     return [basic_image_tif] * 4
 
 
+@pytest.fixture
+def in_paths_mismatch(basic_image_tif, basic_image_tif_2):
+    """ Input file paths for tifs of different sizes to stack. """
```

Comment:
Is this different sizes or different dimensions? ie the shape is different?

Reply:
Correct, the shapes of basic_image_tif and basic_image_tif_2 are different (10x10 and 20x20, respectively).